lms/urls.py

```diff
@@ -98,7 +98,7 @@
     url(r'', include('openedx.features.philu_courseware.urls')),
     url(r'', include('openedx.features.badging.urls')),
     url(r'', include('openedx.features.specializations.urls')),
-    url(r'^marketplace/', include('openedx.features.marketplace.urls')),
+    url(r'^communityhub/', include('openedx.features.marketplace.urls')),
```

Comment:
How about if we only change marketplace to communityhub (just the endpoint) here instead of doing it at all places?
As still our app name is also marketplace, means for our code it is marketplace. 



WDYT?
@ahmed-faraz46 @aliassad

Reply:
Even our template names are like **marketplace_listing.html** 
Either we should change all these things as well to be consistent. But to keep it simple I think we can revert the url names and just change the endpoint from marketplace to communityhub.

Reply:
yes you are right change only the URL to communityhub and reverting other changes.

Reply:
I do agree with you and @monofal as well, either we should also change the name of the app or we should just rename the main url to `communityhub` instead of renaming all url names and their usages.